### PR TITLE
simplify kubecost instructions

### PR DESCRIPTION
**Pull request by jessegoodier**
* This is a new doc and I was asked to review it. Some minor changes to make it easier to follow.
Please let me know if you have any questions on it. I have tested the instructions, and they work. 
Jesse @ kubecost

### CODE REVIEW

**Comment by bradleycamacho** [comment]
Hi @jessegoodier, thanks for the edit! I'll take a look and get this merged in. 

Before I can do that though, could you take a second to sign the CLA by clicking the yellow button above this comment? Thanks!

**Comment by jessegoodier** [comment]
> Hi @jessegoodier, thanks for the edit! I'll take a look and get this merged in.
> 
> Before I can do that though, could you take a second to sign the CLA by clicking the yellow button above this comment? Thanks!

done!

File: src/content/docs/kubernetes-pixie/kubecost/get-started-kubecost.mdx [hunk 1]
```diff
@@ -14,20 +14,21 @@ With New Relic and [Kubecost](https://kubecost.github.io/cost-analyzer/), you ca
...
+* What are the cost breakdowns by namespace for my application workloads?
 * How can I tailor cost estimation based on the pricing/discounts provided to me by my cloud providers?
 
 ## Get started
 
-To get started, first set up [Prometheus Remote Write](/docs/infrastructure/prometheus-integrations/install-configure-remote-write/set-your-prometheus-remote-write-integration/) in New Relic, then install the Kubecost agent.
+This guide assumes that you will use the Kubecost bundled Prometheus server. It is possible to use your own, though it is outside the scope of this guide.
 
-### Set up Prometheus Remote Write
+You will need to create a `Remote Write` endpoint in New Relic, which you can do by clicking the green "Add Prometheus Data" botton on this page. Note that the bearer token will be used.
```
Comment (brettj-newrelic) [inline src/content/docs/kubernetes-pixie/kubecost/get-started-kubecost.mdx:24]: `button` is spelled incorrectly

**Comment by jessegoodier** [comment]
@brettj-newrelic can you review again, sorry I missed this notification and it fell off my radar.

**Review by brettj-newrelic — APPROVED** [review]

**Review by ally-sassman — APPROVED** [review]

**Pull request merged**